### PR TITLE
Refactor duplicated code of creation of SQS object into a function

### PR DESCRIPTION
**Pull request by nsjcorps**
I refactored the duplicated code of creation of SQS object into a function as requested om the GCI task.

I created a function to handle the queue_name that was previously not following DRY principle.

I then imported into all the files that needed it.

### CODE REVIEW

**Comment by nsjcorps** [comment]
I have made the changes for the task. However, I can see it was failing already from some changes made yesterday by @RishabhJain2018 

@Ram81 , @vkartik97 , @RishabhJain2018  Please review.

File: apps/jobs/sender.py [hunk 1]
```diff
@@ -55,24 +37,27 @@ def get_or_create_sqs_queue(queue_name):
...
+        submission_pk: Submission Id
 
     Returns:
         Returns SQS response
     """
+    message = {
+        "challenge_pk": challenge_pk,
+        "phase_pk": phase_pk,
+        "submission_pk": submission_pk,
+    }
```
Comment (vkartik97) [inline apps/jobs/sender.py]: Hey,
I didn't get the reason for this change

**Comment by nsjcorps** [comment]
@vkartik97

That was not my change. I just merged it with the master branch

**Comment by nsjcorps** [comment]
I will revert it though

**Comment by nsjcorps** [comment]
I don't know how that happened but it has been fixed @vkartik97

**Comment by nsjcorps** [comment]
Sorry about that

**Pull request closed without merging**